src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -1077,4 +1077,13 @@ Do you want to proceed?</value>
   <data name="All_methods" xml:space="preserve">
     <value>All methods</value>
   </data>
+  <data name="Inline_Diagnostics_Error" xml:space="preserve">
+    <value>Inline Diagnostics - Error</value>
```

Comment:
if this is our user string, teh namespaces/etc. for this feature should be called `Inline Diagnostics` not `Inline Errors`